data-raw/healthy-places/access-to-green-space/access-greenspace.R

```diff
@@ -0,0 +1,25 @@
+# Load packages
+library(tidyverse)
+library(geographr)
+library(devtools)
+library(styler)
```

Comment:
Please remove styler package, it is typically loaded in your personal session rather than the script